examples/third_party_examples/ai_agent_functions_node.md

```diff
@@ -0,0 +1,359 @@
+# Build an AI agent with OpenAI functions and the Node.js SDK
```

Comment:
Oh an let's title the file like the others, with sentence case, we can also move it outside of `third_party_examples`, that folder is for examples that use other companies/products on top of OpenAI. 

`How_to_build_an_agent_with_the_node_sdk.ipynb`

Reply:
Got, will fix these things now!

Reply:
Btw, I used the `.md` file extension initially. Should I change it to `.ipynb` even though it only consists of markdown?

Reply:
Oh my bad, yeah `.md` is correct